plugins/parsers/csv/parser_test.go

```diff
@@ -375,15 +398,52 @@ func TestSkipColumnsWithHeader(t *testing.T) {
 	)
 	require.NoError(t, err)
 	testCSV := `col,col,col
-	1,2,3
-	trash,80,test_name`
+1,2,3
+trash,80,test_name`
```

Comment:
Why is this necessary?

Reply:
Can you please comment on why the reformatting is necessary?

Reply:
I'm ok with removing them but they might be used to test the trim-space feature... If you don't mind I would feel better to keep them...

Reply:
Let's not remove the space. I don't see a reason to remove it.